packages/apps/kadena-transfer/package.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Can we add the `lint-staged`

[example](https://github.com/kadena-community/kadena.js/blob/d7ea6eb7e343c79532abe791b25ade2d530a0863/packages/apps/kadena-docs/package.json#L22-L30)